### DISCUSSION TITLE
Characters with pressure stabilizer consume oxygen from suits

### DISCUSSION THREAD

**Discussion by Xeavas** [FakeFishGames/Barotrauma#15097] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Characters with pressure stabilizer consume oxygen from oxygen tanks in diving suits.

### Reproduction steps

1. Use pressure stabilizer
2. Equip suit with full oxygen tank

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Single player

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Windows

**Comment by Regalis11** [comment]
Thank you for the report! I would not necessarily consider this a bug: pressure stabilizer makes you survive without oxygen, but nothing in the description implies that it makes you stop breathing entirely.
